02-Git/main.cpp

```diff
@@ -33,6 +33,14 @@ const char * bot_picks() {
     return game_options[ rand() % NUMBER_OF_GAME_OPTIONS ].short_option;
 }
 
+const char * get_option_name_by_code(const char *option_code) {
```

Comment:
This comment is not for this line but to commit message:
Commit title should be 02-Git: Improve GUI loop